### PR TITLE
adding support for RDS Proxies

### LINKED ISSUE #564: Support Amazon RDS db-proxy
**Describe the solution you'd like**
Collect Amazon RDS db-proxy
**Additional context**
To get an idea of how to implement the collector, take the [RDS Instances](https://github.com/tailwarden/komiser/blob/master/providers/aws/rds/instances.go) resource as inspiration.

### PR DESCRIPTION
**Pull request by AnnuCode**
## Problem

fixes #564 

## Solution

Adding support to collect Amazon RDS db-proxy.

## Changes Made

- added `proxies.go` file in the `rds` folder.

## Notes

- want to confirm if we can `tag` the RDS Proxy, this [link](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Tagging.html) provides more info.
- want to find the correct format for `Link` in the `Resources` slice.

## Checklist

- [ ] Code follows the <a href="https://github.com/tailwarden/komiser/blob/master/CONTRIBUTING.md">contributing</a> guidelines
- [ ] Changes have been thoroughly tested
- [ ] <a href="https://github.com/tailwarden/docs.komiser.io">Documentation</a> has been updated, if necessary
- [ ] Any dependencies have been added to the project, if necessary

## Reviewers

@[username of the reviewer]

### CODE REVIEW

**Comment by mlabouardy** [comment]
@AnnuCode do you need some help for this one? :)

**Comment by AnnuCode** [comment]
@mlabouardy Hi, can you please share the correct format of  RDS Proxy url to be used as `LINK` in the `resources` slice?

**Comment by mlabouardy** [comment]
> @mlabouardy Hi, can you please share the correct format of RDS Proxy url to be used as `LINK` in the `resources` slice?

Hey @AnnuCode, the format is the following:
`https:/REGION.console.aws.amazon.com/rds/home?region=REGION#proxies:PROXY_NAME`

File: providers/aws/rds/proxies.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+				hourlyUsage = int(time.Since(*proxy.CreatedDate).Hours())
+			}
+
+			hourlyCost := 0.0
+			monthlyCost := float64(hourlyUsage) * hourlyCost
+
+			resources = append(resources, models.Resource{
+				Provider:   "AWS",
+				Account:    client.Name,
+				Service:    "RDS Instance",
```
Comment (mlabouardy) [inline providers/aws/rds/proxies.go:48]: should be a "RDS Proxy"

File: providers/aws/rds/proxies.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+			break
+		}
+
+		config.Marker = output.Marker
+	}
+	log.WithFields(log.Fields{
+		"provider":  "AWS",
+		"account":   client.Name,
+		"region":    client.AWSClient.Region,
+		"service":   "RDS Instance",
```
Comment (mlabouardy) [inline providers/aws/rds/proxies.go:68]: "RDS Proxy"

**Review by mlabouardy — CHANGES_REQUESTED** [review]
Minor change on copy :)

**Review by mlabouardy — APPROVED** [review]
LGTM!

**Pull request merged**